src/import.cc

```diff
@@ -128,6 +128,9 @@ AbstractNode *ImportModule::instantiate(const Context *ctx, const ModuleInstanti
 	node->origin_x = node->origin_y = 0;
 	origin->getVec2(node->origin_x, node->origin_y);
 
+	auto center = c.lookup_variable("center", true);
+	node->center = center->toBool();
```

Comment:
In other parts of the code we do a type check before converting:
https://github.com/openscad/openscad/blob/707043efe658e9bf41ea102fafad1c093c7fdbfd/src/primitives.cc#L199
https://github.com/openscad/openscad/blob/707043efe658e9bf41ea102fafad1c093c7fdbfd/src/primitives.cc#L210-L212
I am not sure what that implies, I just see a difference in handling the same variable.

Reply:
No idea, might be some leftover from experimenting with things. Makes sense to have it using the same logic as in other places.